queries.sql

```diff
@@ -15,3 +15,44 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE  name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE  species IS NULL;
+COMMIT;
+SELECT * FROM animals;
```

Comment:
Hi @OybekKayumov 
### **OPTIONAL**
These changes are part of the project requirement but I mark it optional because I find it to be the only worry you have for this Pull Request. Please kindly do the changes before merging this PR.

- [ ] Your query here is selecting all from the animals table instead of selecting just species `SELECT species from animals;`  to verify that change was made

Reply:
OK